src/extensions/starter_dashlet/BoxWithHandle.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { CSSProperties, FC, memo } from 'react';
+import { useDrag } from 'react-dnd';
+import Icon from '../../controls/Icon';
+import { StarterInfo } from '../../util/api';
+
+interface IProps {
+  children?: any;
+  className?: string;
+  item: StarterInfo;
+}
+
+export const BoxWithHandle: FC<IProps> = memo((props: IProps) => {
+  const [{ opacity, isDragging }, drag, dragPreview] = useDrag({
+    item: { id: props.item.id, type: 'TOOL' },
+    collect: (monitor) => {
+      return {
+        isDragging: monitor.isDragging(),
+        opacity: monitor.isDragging() ? 0.4 : 1,
+      };
+    },
+  });
+  const children = Array.isArray(props.children)
+    ? props.children : [props.children];
+  return (
+    <div className='starter-drag-handle-container' ref={dragPreview} style={{ opacity }}>
```

Comment:
"BoxWithHandle" is a fairly generic name and this component seems to be quite generic but these class names are very specific